tests/testthat/test-parsers.R

```diff
@@ -8,169 +8,169 @@ test_that("parsers throw on invalid tz argument", {
 })
 
 test_that("ymd functions correctly parse dates separated by -", {
-  expect_that(ymd("2010-01-02"),
-              equals(as.Date("2010-01-02")))
-  expect_that(ymd("10-01-02"),
-              equals(as.Date("2010-01-02")))
-  expect_that(ydm("2010-02-01"),
-              equals(as.Date("2010-01-02")))
-  expect_that(mdy("01-02-2010"),
-              equals(as.Date("2010-01-02")))
-  expect_that(myd("01-2010-02"),
-              equals(as.Date("2010-01-02")))
-  expect_that(dmy("02-01-2010"),
-              equals(as.Date("2010-01-02")))
-  expect_that(dym("02-2010-01"),
-              equals(as.Date("2010-01-02")))
-  expect_that(ymd(c("2010-01-02", "2010-01-03")),
-              equals(as.Date(c("2010-01-02", "2010-01-03"))))
+  expect_equal(ymd("2010-01-02"),
```

Comment:
Would you mind reformatting these tests to fit on one line?

Reply:
i did this wherever possible and removed stray blank lines!